### PR TITLE
Many, many updates

### PR DESCRIPTION
**Pull request by forana**
* Added "Concepts" articles. About half are consolidations of what we had before, half are new (including explaining pods, environments, services, jobs, containers, and console).
* Added an article with tips on how to write an application to work on Stratum.
* Consolidated several other topics, including SSL Certs, Buildpacks, Sites, and Organizations.
* Overhauled the Initial Setup guide.
* More linking between articles.
* Fixed many 404s and updated many 301s.

### CODE REVIEW

File: concepts/environments.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Environments
+category: concepts
+summary: Learn what Environments are on Stratum.
+alias: "stratum/articles/environment-defining/"
+---
+
+**Environments** are the highest-level object that a user can interact with on [Stratum](https://catalyze.io/stratum). An Environment represents your entire hosted application or product - runtimes, databases, caches, queues, tools, and all. If you have both production and staging hosted through
```
Comment (catalyzeVince) [inline concepts/environments.md]: > An Environment represents your entire hosted application or product
I really want to take Environments down a different path for public explanation.

It is a compliant virtual datacenter or something like that.

I think we should hold off on this article until we can touch on all the bases like SSDN, etc.

Comment (forana) [inline concepts/environments.md]: I'll hold off on touching that article.

File: initial-setup.md [hunk 2]
```diff
@@ -4,146 +4,163 @@ category: getting-started
...
 
-**Recap:** Create a site in your environment using the certificate and DNS name you want to resolve to your Stratum application
+```
+git push catalyze master
+```
 
-## Push Code
+> ***Note:*** The pushed branch **must** be `master`. If the branch you want to push is not `master`, use the following:
+>
+> ```
```
Comment (rr1000) [inline initial-setup.md]: Any reason to have the code included in the blockquote here? Or use a block quote at all?

Comment (forana) [inline initial-setup.md]: the code is part of the note. using blockquotes to call out notes.

**Pull request merged**